snowex_db/upload/batch.py

```diff
@@ -7,9 +7,60 @@
 from os.path import abspath, basename, expanduser, join
 
 from snowex_db.interpretation import get_InSar_flight_comment
-from snowex_db.metadata import (DataHeader, read_InSar_annotation)
+from snowex_db.metadata import (ExtendedSnowExMetadataParser, read_InSar_annotation)
 from snowex_db.upload.rasters import UploadRaster
-from snowex_db.utilities import get_logger
+from snowex_db.utilities import get_logger, metadata_to_dict
+from snowexsql.db import get_table_attributes
+from snowexsql.tables import Site
+from snowex_db.projection import add_geom
+
+
+class SiteDetailsUploader:
```

Comment:
I think this may be handled completely by the layer uploader now. We may just be able to get rid of this functionality